_includes/ext-link-marker.html

```diff
@@ -0,0 +1 @@
+<span style="font-size: 0.8em;display:inline-block;margin-left: 4px"><i class="fa-solid fa-up-right-from-square"></i></span>
```

Comment:
Could we move the style here to the CSS file? Maybe under `.navar-link` and `.navbar-item`.

Reply:
Not really seeing any advantage to doing that. These styles are never used anywhere else, and they'll get lost in the 1000 line css file. This component (as it exists now) could also be used in places other than the navbar.